.github/CONTRIBUTING.md

```diff
@@ -164,16 +164,23 @@ You will find the built documentation in `docs/_build/html`.
 
 - If you add a new feature, demonstrate its awesomeness on the [examples page](https://github.com/python-attrs/attrs/blob/main/docs/examples.md)!
 
-- For docstrings, we follow [PEP 257](https://peps.python.org/pep-0257/) and use the `"""`-on-separate-lines style:
+- For docstrings, we follow [PEP 257](https://peps.python.org/pep-0257/), use the `"""`-on-separate-lines style, and [Napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html)-style API documentation:
 
   ```python
-  def func(x):
+  def func(x: str, y: int) -> str:
       """
       Do something.
 
-      :param str x: A very important parameter.
+      Args:
+          x: A very important argument.
 
-      :rtype: str
+          y:
+            Another very important argument, but its description is so long
+            that it doesn't fit on one line. So, we start the whole block on a
+            fresh new line to keep the block together.
+
+      Returns:
```

Comment:
It'd be useful to show that raises/yields exist too.

Reply:
Instead of `Returns` you can also use `Return`, which makes imho a bit more sense given the fact that docstrings are written in imperative (i.e. "Do something, return something"). 

But this is just a minor nitpick and afaik it is both rendered the same way.

Reply:
Only the summary line needs to be in imperative mood, though. The rest is behavior explanation.
Besides, the rendered HTML will still have “Returns” so why not match it.

Reply:
Yeah, to me it’s different too. The main body says what will happen, the rest is rather “descriptive”.